airflow/www/templates/airflow/ti_log.html

```diff
@@ -123,8 +123,32 @@ <h4>{{ title }}</h4>
             if(auto_tailing && checkAutoTailingCondition()) {
               var should_scroll = true
             }
+            
+            // ANSI color support
+            var ansi_color_regex = /.\[([0-9][0-9])m(.*?).\[0m/gs;
+            var res.message = res.message.replace(ansi_color_regex, function(match, color, content) {
+              var colors = {
+                30: 'black',  40: 'black',
+                31: 'red',    41: 'red',
+                32: 'green',  42: 'green',
+                33: 'yellow', 43: 'yellow',
+                34: 'blue',   44: 'blue',
+                35: 'magenta',45: 'magenta',
+                36: 'cyan',   46: 'cyan',
+                37: 'white',  47: 'white'
+              }
+
+              if (30 <= parseInt(color) && parseInt(color) <= 37) {
```

Comment:
Will it work if the text is multi-colored?  What do you think about using [ansi_up.js](https://github.com/drudru/ansi_up) (License: MIT - Apache compatible) ?

Reply:
+1 for using existing solution

Reply:
I tested the change in a local environment with ansi_up and it's working like a magic :-)
The problem is that I don't know where I should inclute the new js file.
Some help please

Reply:
Hi.
I fond the right location, in the static directory.

Reply:
The CI check failed because of there is no Apache header. This is a problem?
The ansi_up library is MIT licensing

Reply:
@mik-laj can you advise?

Reply:
This looks good for now. In the future, I think we should move the JS code to separate files and then download this library using npm and webpack.